docs/sources/flow/reference/components/pyroscope.ebpf.md

```diff
@@ -0,0 +1,178 @@
+---
+title: phlare.ebpf
+labels:
+  stage: beta
+---
+
+# phlare.ebpf
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`pyroscope.ebpf` configures an ebpf profiling job for the current host. The collected performance profiles are forwarded
+to
+the list of receivers passed in
+`forward_to`.
+
+Multiple `pyroscope.ebpf` components can be specified by giving them different labels, however it is not recommended.
+
+## Usage
+
+```river
+pyroscope.ebpf "LABEL" {
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+
+The component configures and starts a new ebpf profiling job to collect performance profiles from the current host.
+
+The following arguments can be used to configure a `pyroscope.ebpf`. Only the
+`forward_to` field is required and any omitted fields take their default
+values.
+
+| Name                      | Type                     | Description                                                            | Default | Required |
+|---------------------------|--------------------------|------------------------------------------------------------------------|---------|----------|
+| `forward_to`              | `list(ProfilesReceiver)` | List of receivers to send collected profiles to.                       |         | yes      |  
+| `targets`                 | `list(map(string))`      | List of targets to group profiles by container id                      |         | no       |   
+| `default_target`          | `map(string)`            | Default target to use when a PID is not associated with a container id |         | no       |    
```

Comment:
I wonder if we should get rid of this and just drop those. I'd rather defer this to another service discovery component ? WDYT ?

Reply:
This would also allow to get rid of `targets_only`, less config option the better.

Reply:
I agree it is confusing and I needed it mostly for local development.
I removed both default_targets and targets_only

Reply:
Hmm, actually I found the feature to be very useful. What about removing the documentation and keeping the options until something like `discovery.whole_system` is available?